commands/command_filter.go

```diff
@@ -49,7 +61,10 @@ func clean(reader io.Reader, fileName string) ([]byte, error) {
 	if errors.IsCleanPointerError(err) {
 		// TODO: report errors differently!
 		// os.Stdout.Write(errors.GetContext(err, "bytes").([]byte))
-		return errors.GetContext(err, "bytes").([]byte), nil
+		// return errors.GetContext(err, "bytes").([]byte), nil
+
+		// TODO(taylor): what does this mean?
```

Comment:
👍  this came up during our pairing and was addressed in https://github.com/github/git-lfs/pull/1622/commits/1d2b5ce985cb508c9d25ab5029561227e66d7b9f.